### PR TITLE
✨ Tailwind configs w/ KF theme

### PR DESCRIPTION
**Pull request by bdolly**
Rolls theme colors and typography into the tailwind config. To accomplish this the build process had to be hacked to the following process: 
1. the tailwind config file is compiled with babel into `.tmp/tailwind.babel.js`, this is to allow the use of ES6 features such as `import` and keep from mixing `import` and `require` in the same file 
2. compile the components and theme to `/dist` so that the tailwind.babel.js file can import them 
3. compile and build the main stylesheet with webpack that loads in the compiled `.tmp/tailwind.babel.js` config file

### CODE REVIEW

**Comment by dankolbman** [comment]
Can you get rid of those old commits for styled-system?

**Comment by bdolly** [comment]
@dankolbman netlify is failing because I didn't update the storybooks with the new css and code structure, but other than that this should be good to go

File: tailwind.config.js [hunk 1]
```diff
@@ -42,41 +50,127 @@ View the full documentation at https://tailwindcss.com.
...
-  'error-light': '#fadfe1',
-  'warn': chroma(brand.primary).brighten(1).css(),
-  'highlight': '#e83a9c',
-  'active': '#00afed',
-  'inactive': '#dedfe4',
-  'optionSelected': '#e5f6fd'
-}
+  white: '#ffffff',
+
+  'red-darkest': '#3b0d0c',
```
Comment (dankolbman) [inline tailwind.config.js]: Do you want to keep all the tailwind colors here?
Maybe better to replace with our own and use chroma if we want shades too.

File: tailwind.config.js [hunk 2]
```diff
@@ -42,41 +50,127 @@ View the full documentation at https://tailwindcss.com.
...
+  'pink-dark': '#eb5286',
+  pink: '#f66d9b',
+  'pink-light': '#fa7ea8',
+  'pink-lighter': '#ffbbca',
+  'pink-lightest': '#ffebef',
+};
+
+const fonts = {
+  ...theme.Typography.fonts,
+  sans: [
```
Comment (dankolbman) [inline tailwind.config.js]: Do we need these font stacks if we are using the above?

File: tailwind.config.js [hunk 3]
```diff
@@ -268,12 +306,12 @@ module.exports = {
   */
 
   tracking: {
-    'tight': '-0.05em',
-    'normal': '0',
-    'wide': '0.05em',
+    tight: '-0.05em',
+    normal: '0',
+    wide: '0.05em',
+    ...theme.Typography.letterSpacing,
```
Comment (dankolbman) [inline tailwind.config.js]: Should this replace the above?

File: src/theme/Typography.js [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+  text-decoration: none;
+  border: none;
+`;
+
+const headingsBase = `
+  font-family: ${fonts.montserrat};
+  ${typographyBase}
+`;
+
+export const p = `
```
Comment (dankolbman) [inline src/theme/Typography.js]: These css styles should be put in a css file instead of the configuration.

**Comment by bdolly** [comment]
closing as this approach was abandoned due to complexity

**Pull request closed without merging**